beacon-chain/blockchain/kzg/kzg.go

```diff
@@ -28,19 +28,15 @@ const BytesPerCell = ckzg4844.FieldElementsPerCell * ckzg4844.BytesPerFieldEleme
 // BytesPerBlob is the number of bytes in a single blob.
 const BytesPerBlob = ckzg4844.BytesPerBlob
 
-// FieldElementsPerCell is the number of field elements in a single cell.
-// TODO: This should not be exposed.
-const FieldElementsPerCell = ckzg4844.FieldElementsPerCell
+// fieldElementsPerCell is the number of field elements in a single cell.
+const fieldElementsPerCell = ckzg4844.FieldElementsPerCell
```

Comment:
This is no longer exposed because callers no longer need to flatten/deflatten the cells